README.md

```diff
@@ -198,3 +198,84 @@ func Function(
 	
 }
 ```
+
+### Sync Tool and Operator Types
+
```

Comment:
Can you please start with issue description and motivation behind adding this tool.
Basically when project has a lot of dependencies it hard to re-use/import to another projects. We also hit dependencies conflict between operators